app/views/content/life-as-a-teacher/age-groups-and-specialisms/primary/_article.html.erb

```diff
@@ -94,6 +94,9 @@
     <h3>Progression as a primary school teacher<h3>
       <p>As a primary school teacher, there's plenty of opportunity for progression and opportunities to lead beyond the classroom. That could mean becoming a lead in a subject, leading a key stage or even becoming a part of the senior management team.</p>
      <%= render 'content/shared/quotes/quote_primary_phil' %>
+    <h3>Find primary teacher training courses<h3>
+      <p>Explore the different primary courses available.</p>
+      <%= render CallsToAction::ArrowLinkComponent.new(link_target: "https://find-teacher-training-courses.service.gov.uk/results?subjects%5B%5D=00&subjects%5B%5D=01&subjects%5B%5D=02&subjects%5B%5D=03&subjects%5B%5D=04&subjects%5B%5D=06&subjects%5B%5D=07&applications_open=true&subject_name=&subject_code=&location=&radius=10&provider_name=&provider_code=&order=course_name_ascending", link_text: "Find your primary teacher training course") %>
```

Comment:
```suggestion
      <%= render CallsToAction::ArrowLinkComponent.new(link_target: "https://find-teacher-training-courses.service.gov.uk/results?subjects%5B%5D=00&subjects%5B%5D=01&subjects%5B%5D=02&subjects%5B%5D=03&subjects%5B%5D=04&subjects%5B%5D=06&subjects%5B%5D=07&applications_open=true&order=course_name_ascending", link_text: "Find your primary teacher training course") %>
```
I think we can take some of the unused parameters out of the link?

Reply:
Alternatively, if you mention specialisms here, you could direct them to `https://find-teacher-training-courses.service.gov.uk/primary` ? This might be cleaner and less likely to change?